docs/conf.py

```diff
@@ -227,20 +227,18 @@ def generate_r_docs(app):
     echo 'options(repos = "https://cran.rstudio.com")' > $HOME/.Rprofile
     /home/docs/.conda/bin/conda create -q -y -n r_env \
         r-base=3.5.1=h1e0a451_2 \
-        r-devtools=1.13.6=r351h6115d3f_0 \
         r-jsonlite=1.5=r351h96ca727_0 \
         r-matrix=1.2_14=r351h96ca727_0 \
         r-testthat=2.0.0=r351h29659fb_0 \
-        cmake=3.14.0=h52cb24c_0 \
-        ca-certificates=2019.11.27=0
+        cmake=3.14.0=h52cb24c_0
     /home/docs/.conda/bin/conda install -q -y -n r_env -c conda-forge \
         r-data.table=1.12.8=r35hcdcec82_0 \
-        r-pkgdown=1.3.0=r35h6115d3f_1001 \
-        r-roxygen2=6.1.1=r35h0357c0b_1001
+        r-pkgdown=1.5.1=r35h6115d3f_0 \
+        r-roxygen2=7.1.0=r35h0357c0b_0
     source /home/docs/.conda/bin/activate r_env
     export TAR=/bin/tar
     cd {0}
-    sed -i'.bak' '/# Build the package (do not touch this line!)/q' build_r.R
```

Comment:
OK, indeed `roxygenize(load = "installed")` results in that LightGBM library is built only once! Great! Please remove this creepy comment in the source file then 😃 .
```
(do not touch this line!)
```

Reply:
oh right! yep I'll remove that